monai/handlers/lr_schedule_handler.py

```diff
@@ -47,6 +48,9 @@ def __init__(
                 `True` is epoch level, `False` is iteration level.
             step_transform: a callable that is used to transform the information from `engine`
                 to expected input data of lr_scheduler.step() function if necessary.
+            logger_handler: if `print_lr` is True, add additional handler to log the learning rate: save to file, etc.
+                all the existing python logging handlers: https://docs.python.org/3/library/logging.handlers.html.
+                the handler should have a logging level of at least `INFO`.
```

Comment:
is this line 53 copied here by mistake?

Reply:
Hi @wyli ,

It's not a mistake, the `logger_handler` should take level > INFO as we use `logger.info` to save:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/handlers/lr_schedule_handler.py#L85

Thanks.